### DISCUSSION TITLE
Does CredentialsManager work on Linux?

### DISCUSSION THREAD

**Discussion by buptwsg** [fastlane/fastlane#17603] [Q&A]
Hi there,
I want to know if CredentialsManager works on Linux platform. From the document, credentials are added to Keychain. So I'm not sure if it can work on Linux.

Please help me, thanks!

[a comment by an automated account is not shown]

**Comment by buptwsg** [comment]
hi, I'm investigating, but I don't have a Linux environment. If I have, I can try it myself. I just want to know if I can use CredentialsManager to save account and password on Linux platform.

Thanks.

> It seems like you have not included the output of `fastlane env`
> 
> To make it easier for us help you resolve this issue, please update the issue to include the output of `fastlane env` 👍

**Comment by rogerluan** [comment]
Duplicate of https://github.com/fastlane/fastlane/discussions/17598
